grants_tagger/__main__.py

```diff
@@ -352,10 +352,16 @@ def tag(
         tagged_grants_path: Path = typer.Argument(..., help="path to output csv"),
         model_path: Path = typer.Argument(..., help="path to model"),
         label_binarizer_path: Path = typer.Argument(..., help="label binarizer for Y"),
-        threshold: float = typer.Option(0.5, help="threshold upon which to assign tag")):
+        approach: str = typer.Argument(..., help="approach used to train the model"),
+        threshold: float = typer.Option(0.5, help="threshold upon which to assign tag"),
+        grant_id_field: str = typer.Option("grant_id", help="field name for grant id"),
+        grant_text_fields: str = typer.Option("title,synopsis", help="comma separated text fields to be used for tagging"),
+        text_null_value: str = typer.Option("No Data Entered", help="value indicating null in text fields")):
```

Comment:
It would probably need a bit of re-factoring, but there is also a lot of junk synopses that @lizgzil and I collected over time to clean before training models: 

https://github.com/wellcometrust/datalabs/blob/master/modelling/grants/cleaning/common.json

Might be interesting to try and add to your pipeline at some point.